janusgraph-backend-testutils/src/main/java/org/janusgraph/graphdb/GraphDatabaseConfigurationInstanceIdTest.java

```diff
@@ -100,4 +103,19 @@ public void instanceIdShouldEqualHostnamePlusSuffix() throws UnknownHostExceptio
         assertEquals(graph.openManagement().getOpenInstances().toArray()[0], Inet4Address.getLocalHost().getHostName() + "1");
 				graph.close();
     }
+
+    @Test
+    public void shouldCreateCustomBackendThreadPoolSize() throws UnknownHostException {
```

Comment:
Are you able to run this test locally? I got `java.lang.IllegalArgumentException: Could not find implementation class: org.janusgraph.diskstorage.inmemory.InMemoryStoreManager` when I try running this test class. I suspect it is not picked up by CI at all.

Reply:
You are right. Thank you for this catch! I will fix it in a separate PR

Reply:
I guess you will write a test in another place, e.g. JanusGraphTest.java, right?

Reply:
Kind of yes. I'm working on it right now but it uncovered some mistakes in these tests and a bug. Fixing it. I will provide a PR today

Reply:
Oh sorry. No, I will not write a test in a separate place. I want first fix these tests and then rebase this PR

Reply:
Got it, that sounds fair

Reply:
Fixed tests here #2697 except the one which is now tracked here #2696

Reply:
@li-boxuan I have rebased this PR against the latest master commit, so this test should run now